demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -24,17 +28,20 @@ class Runner:
     ERROR_ENTRY_TYPE = 4
     DEBUG_FILE_ENTRY_TYPE = 16
     SECTIONS_HEADER_REGEX = re.compile(r'^(Context Outputs|Human Readable section|Raw Response section)')
+    RAW_RESPONSE_HEADER = re.compile(r'^Raw Response section')
```

Comment:
do we want to try extracting the data from a `Context Outputs` section first? as this is a better representation of the command output (which may differ from the raw response)?
if not available (or empty) we can default to the `Raw Response` section.

Reply:
@ronykoz What do you think? As you were the one who specified this task.

Reply:
Will do